website/about.html

```diff
@@ -113,86 +113,104 @@ <h2>Why We Created This Project</h2>
   <div class="row">
     <div class="column">
       <div class="card">
-        <img src="https://astrochart.github.io/website/team_members/Bry.JPG" class="center" alt="Bryanna" style="width:80%;">
+        <img src="team_members/Dylan.jpeg" class="center" alt="Dylan" style="width:80%;">
+        <div class="container">
+          <h2>Dylan Gappa</h2>
+          <p class="title">Simulation</p>
+          <p>Dylan is using existing professional surveys to create a simulator for CHART.
+              When completed, users will be able to create a mock data set
+              to match when and where they observed.</p>
```

Comment:
Future goals are to start a career in the civilian service of the military, working on aerospace technology.